### ISSUE TITLE
Test error -  T5EncoderModel._load_pretrained_model()

### ISSUE THREAD

**Issue by venturamor** [martiansideofthemoon/rankgen#9]
Hi,
just tried to run the test as in the ReadMe instructions and got an error - 
line 45 in t5x_embeddings.py:        
self.model, _, _, _, _ = T5EncoderModel._load_pretrained_model(model, state_dict_new, None, f"google/t5-v1_1-{self.model_size}")

It worked only when I added [] before the None as the loaded_keys:
self.model, _, _, _, _ = T5EncoderModel._load_pretrained_model(model, state_dict_new, [], None, f"google/t5-v1_1-{self.model_size}")

**Comment by resloved** [comment]
I ran into this same behavior with `transformers==4.20.1`

Looks like `_load_pretrained_model` was changed with the [v4.19.0](https://github.com/huggingface/transformers/releases/tag/v4.19.0) release as a result of [#16657](https://github.com/huggingface/transformers/pull/16657). Would be nice if there were some version requirements to replicate your results.

**Comment by martiansideofthemoon** [comment]
Hi @venturamor and @resloved, I fixed this in 89cb98766e9d66454331c643603466b2eed91c91, sorry for the trouble it caused. I've also added a [`requirements.txt`](https://github.com/martiansideofthemoon/rankgen/blob/master/requirements.txt) file for reproducibility, it has the latest HuggingFace version.

Since this solution is not sustainable (HuggingFace may keep changing their internal API), I've added issues #10 and #11 to keep track of this.
